### ISSUE TITLE
Standard way of indicating post type

### ISSUE THREAD

**Issue by njkleiner** [indieweb/microsub#43]
The Microsub specification currently does not define a standard way of including an item's [post type](https://indieweb.org/post-type-discovery) in a timeline response.

From what I can tell, there are currently some server implementations that include post type in their response, like [Aperture](https://github.com/aaronpk/Aperture) in the `post-type` property, but there does not seem to be consensus on how to do this (mainly due to a lack of implementations).

Wouldn't it make sense to make this part of the spec? I would consider this a part of data normalization/transformation which is literally the purpose of a server.

I suggest we use a top-level field like `_kind` (to match `_id`, `_is_read` and `_source`) as a standard way to indicate post type, `post-type` seems inappropriate since post type is part of an items metadata and not a MF2/JF2 property.

**Comment by jalcine** [comment]
Good point! I'm in favor of using <code>_kind</code> as well since type would be overloaded. This is going to be [implemented in Lighthouse](https://git.jacky.wtf/indieweb/lighthouse/issues/44).

(Originally published at: https://jacky.wtf/2022/4/nu/nu5nq7OBzE7j6yE5RHS9unJA)
